### ISSUE TITLE
Header: About: must add link to /cd-or-ci

### ISSUE THREAD

**Issue by jacobdgm** [DDMAL/CantusDB#1102]
Debra created a new flatpage, moving some content that had previously been on the homepage.

She's requested the link should read "Cantus Database or Cantus Index?".

The page is currently located at https://cantusdatabase.org/about/cd-or-ci/. This path feels adequate, but perhaps not excellent. We have the flexibility to change this before we start linking to it. If anyone has a better idea, please share it.

**Comment by annamorphism** [comment]
What do you find not-excellent about it? The fact that it uses abbreviations like CD and CI? Or that it might be hard to expand to (say) include an explanation of when you might want to use Cantus Ultimus? Would when-to-use be an improvement?

**Comment by jacobdgm** [comment]
Part of it is the initialisms, yes.

Some other things that had crossed my mind: `/cd-and-ci`, `/cd-vs-ci`. Leaning `and`, which feels more cooperative...

**Comment by annamorphism** [comment]
the title of the page is "or" though, and presumably the dropdown menu title will also be "Cantus Database or Cantus Index", so I don't like having the "and" in the path if that's the case.

Personally I would be inclined to revise all of this to incorporate the information about CI on the FAQ page (which *does* say "and"), and maybe replacing that page with this one...So I would say to put this up as is and I will discuss with Debra whether the whole explainer pages need to be revisited more generally.

**Comment by annamorphism** [comment]
I suggested in an email that this text should be dealt with under FAQ, and Debra agreed that this needs reworking anyway (see #738 ). So, Debra intends to merge this text onto the FAQ page herself.  (I think eventually this will want a new name/path, too, but that can be dealt with some other time.)

She also said:
"A link to it might be good on the home page, in addition to the actual CI link. But I don't like the clutter of all the things you could do on both of the sites like it used to be."
So I think we should have "For more information about Cantus Database and Cantus Index, see our FAQ page" right after the link to Cantus Index on the home page.

**Comment by jacobdgm** [comment]
What I'm taking from this: The content on the current cd-or-ci page will probably eventually end up on the FAQ page. So if we create a new link in the header, we'll need to get rid of it again when the cd-or-ci page gets deleted.

I'm going to mark this as "on hold". @annamorphism, if what I said in the paragraph above is substantially correct, can you close this issue?

**Comment by annamorphism** [comment]
@jacobdgm that is basically my understanding, with the addition that I think longterm the FAQ page will probably be renamed something else (possibly something like "cd-or-ci" but possibly something else.) So yes, this is on hold until then, and I will close.
